stl/inc/format

```diff
@@ -2650,6 +2727,45 @@ struct _Arg_formatter {
     }
 };
 
+// Special compile time version of _Parse_format_specs. This version is parameterized on
+// the type of the argument associated with the format specifier, since we don't really
+// care about avoiding code bloat for code that never runs at runtime, and we can't form
+// the erased basic_format_args structure at compile time.

```

Comment:
Should this say `if (!_STD is_constant_evaluated()) { _CSTD abort(); }` to absolutely prohibit runtime invocations?

Reply:
yeah, should we use consteval or mark it with a transition to do so?

Reply:
I'd be ok with `consteval` if it compiles, `TRANSITION` if it doesn't yet compile.